### PR TITLE
line edits to docs in the language directory

### PR DESCRIPTION
**Pull request by christinerose**
Signed-off-by: Christine Rose <professor.rose@gmail.com>

### CODE REVIEW

**Review by shym — COMMENTED** [review]
Thank you for that oldish PR!
I’m porting your suggestions to the current state of the documentation. I left a question about the punctuation of a list.
I’ve seen a couple of edits (in that PR or the others) changing something like “the contract of a function” into “a function’s contract”. For my education, is that just a difference of style or the first one is not grammatically correct? When the possessor is not a person, I never know which one to choose.

**Comment by christinerose** [comment]
> Thank you for that oldish PR! I’m porting your suggestions to the current state of the documentation. I left a question about the punctuation of a list. I’ve seen a couple of edits (in that PR or the others) changing something like “the contract of a function” into “a function’s contract”. For my education, is that just a difference of style or the first one is not grammatically correct? When the possessor is not a person, I never know which one to choose.

It is a matter of style, but I find that too many prepositional phrases make for a clunky, sometimes awkward sentence, so I try to minimise them. Feel free to ignore if you prefer or if it changes the meaning/makes it more confusing.

File: docs/docs/language/function-contracts.md [hunk 1]
```diff
@@ -367,12 +365,12 @@ val inplace_map : ('a -> 'a) -> 'a t -> unit
...
+## Pure Functions
 
-An OCaml function can be declared as `pure`, which means
-- it has no side effect;
-- it raises no exception;
-- it terminates.
+An OCaml function can be declared as `pure`, which means it
+- has no side effect
+- raises no exception
+- terminates
```
[the remark this answers is not in the document]
Comment (christinerose) [reply docs/docs/language/function-contracts.md:373]: Generally items of a list aren't punctuated until they're each a sentence. That said, I think you're correct in this instance! Thanks for pointing that out!

**Comment by shym** [comment]
Thank you for the clarifications and for the update with the punctuation!

**Pull request closed without merging**